### PR TITLE
tests(devtools): add i18n to e2e navigation test

### PR DESCRIPTION
**Pull request by connorjclark**
Couple things need to change upstream, but we'll just handle that on the next roll.

### CODE REVIEW

File: third-party/devtools-tests/e2e/lighthouse/navigation_test.ts [hunk 1]
```diff
@@ -35,9 +62,10 @@ describe('Navigation', async function() {
       });
 
       it('successfully returns a Lighthouse report', async () => {
-        await navigateToLighthouseTab('lighthouse/hello.html');
+        await setDevToolsSettings({language: 'en-XL'});
```
Comment (adamraine) [inline third-party/devtools-tests/e2e/lighthouse/navigation_test.ts:65]: Why modify the existing tests? Couldn't we add another i18n specific test?

Comment (connorjclark) [reply third-party/devtools-tests/e2e/lighthouse/navigation_test.ts:65]: Because these tests are slow and expensive to run.

Comment (adamraine) [reply third-party/devtools-tests/e2e/lighthouse/navigation_test.ts:65]: I think this is better suited for the "test lh with flags changed" test that was added recently. I think we need to do a test sync for that though.

**Review by adamraine — APPROVED** [review]

File: third-party/devtools-tests/e2e/lighthouse/navigation_test.ts (hunk 1 above)
Comment (connorjclark) [reply third-party/devtools-tests/e2e/lighthouse/navigation_test.ts:65]: (spoke offline) we gonna move this around later

**Pull request merged**